initfiles/componentfiles/configschema/xsd/roxie.xsd

```diff
@@ -274,6 +274,9 @@
                     <xs:attribute name="soapTraceLevel" type="xs:nonNegativeInteger" hpcc:displayName="Soap trace Level"
                                   hpcc:presetValue="1"
                                   hpcc:tooltip="Level of detail in reporting SOAPCALL information(set to 0 for none, 1 for normal, >1 or more for extended)"/>
+                    <xs:attribute name="traceTranslations" type="xs:boolean" hpcc:displayName="Trace record layout translations"
+                                  hpcc:presetValue="true"
```

Comment:
Should this tracing be enabled by default?  As it's debug tracing, shouldn't this be disabled by default?

Reply:
It's tracing the users want to look out for... Does that mean it should be a PROGLOG?

I've never been particularly clear on the rules for using the various logging macros.

Reply:
DBGLOG is temporary logging used for debugging/investigation purposes (ithe plan is to have DBGLOG disabled by default.).  For user information, it should be LOG(MCuserInfo, format,...).  There is an explaination of the difference types near the top of jlog.hpp.